### DISCUSSION TITLE
Feature Request: Create our own universal settings

### DISCUSSION THREAD

**Discussion by Ovcheric** [5axes/UniversalCuraSettings#41] [Ideas]
It would be great to set your own custom universal settings. 
It can be like the "Per Model Settings" where you have checkboxes of what parameters to change, and you can set your own values for those parameters. 
Then we can use the universal settings extension for what we need, and override certain parameters with values that best fit to our individual printers. 

Thanks!

**Comment by 5axes** [comment]
I think we can call it a Profile :) Don't really see the benefits in this case
